### PR TITLE
Add debug option to disable media tunneling

### PR DESCRIPTION
**Pull request by Redirion**
#### What is it?
- [ ] Bugfix (user facing)
- [X] Feature (user facing)
- [ ] Codebase improvement (dev facing)
- [ ] Meta improvement to the project (dev facing)

#### Description of the changes in your PR
Added an option for DEBUG builds only to help users to check if their issues with video (blackscreen or stuttering) are a cause of not working media tunneling on their device / custom os.

#### APK testing 
On the website the APK can be found by going to the "Checks" tab below the title and then on "artifacts" on the right.

#### Due diligence
- [X] I read the [contribution guidelines](https://github.com/TeamNewPipe/NewPipe/blob/HEAD/.github/CONTRIBUTING.md).

### CODE REVIEW

File: app/src/main/res/values/settings_keys.xml [hunk 1]
```diff
@@ -172,6 +172,7 @@
     <string name="show_memory_leaks_key" translatable="false">show_memory_leaks_key</string>
     <string name="allow_disposed_exceptions_key" translatable="false">allow_disposed_exceptions_key</string>
     <string name="show_original_time_ago_key" translatable="false">show_original_time_ago_text_key</string>
+    <string name="disable_media_tunneling_key" translatable="false">disable_media_tunneling_text_key</string>
```
Comment (TobiGr) [inline app/src/main/res/values/settings_keys.xml:175]: What's the reason for using a different value here?
```suggestion
    <string name="disable_media_tunneling_key" translatable="false">disable_media_tunneling_key</string>
```

Comment (Redirion) [reply app/src/main/res/values/settings_keys.xml:175]: was asking that myself. I was just copying from  `<string name="show_original_time_ago_key" translatable="false">show_original_time_ago_text_key</string>`

Comment (Redirion) [reply app/src/main/res/values/settings_keys.xml]: I have removed it from both

Comment (TobiGr) [reply app/src/main/res/values/settings_keys.xml]: if you change the key value, we might want to add a SettingsMigration. I am  not sure if this a bit too much effort for a rarely used debug setting.

Comment (Redirion) [reply app/src/main/res/values/settings_keys.xml]: yes, I would say that a migration is not required. But out of curiosity, is there a concept for it in Android and/or do we already have a settings migration in NewPipe?

Comment (TobiGr) [reply app/src/main/res/values/settings_keys.xml]: We already have one: [SettingsMigrations.java](https://github.com/TeamNewPipe/NewPipe/blob/dev/app/src/main/java/org/schabi/newpipe/settings/SettingMigrations.java)

**Review by TobiGr — APPROVED** [review]

**Pull request merged**